lib/src/package_analyzer.dart

```diff
@@ -104,6 +104,9 @@ class PackageAnalyzer {
     InspectOptions options, {
     Future<void> Function(String filename, Uint8List data)? storeResource,
   }) async {
+    const hasToolFailureTag = 'has:tool-failure';
+    const hasAnalysisFailureTag = 'has:analysis-failure';
```

Comment:
Aren't these the same to end-users?

Can we just expose them as one tag?

Reply:
For the end user we could present both as "Platform classification failed because `pana` encountered issues during package analysis." However, for us, it has a small value to separate the two, as `has:tool-failures` can be used to check failures that we are not aware of, and should be fixed.

Reply:
Done.